### PR TITLE
Restrict profile bio length (#156)

### PR DESCRIPTION
**Pull request by altjohndev**
# Solution performed

- [x] Add a counter below the textarea, according to the guidelines from [design](https://www.figma.com/file/vqHVOvugUksHKlhJvNfaFq/LoginCreateAccount-desk?node-id=1%3A17378).

  - Add a generic translation. Id: `edit.input_counter`
  - Add a generic CSS style. Class: `input-counter`
  - Add `textr_` function helper to translate with non-static data

- [x] Restrict the maximum input length to `255`.

## Screenshots

cambiatus_frontend_156_input
> Input length: 4

cambiatus_frontend_156_input_full
> Input length: 255

cambiatus_frontend_156_window
> Profile update page

### CODE REVIEW

File: src/elm/Page/Profile.elm [hunk 1]
```diff
@@ -330,10 +333,19 @@ viewForm loggedIn isDisabled profile avatarS form =
                                 , value form.bio
                                 , disabled isDisabled
                                 , plcH "profile.edit.placeholders.bio"
+                                , maxlength 255
                                 ]
                                 []
                             , viewFieldError "bio" form.errors
                             ]
+                        , div [ class "input-counter" ]
```
Comment (lucca65) [inline src/elm/Page/Profile.elm]: @altjohndev please use tailwind classes instead of raw CSS

File: src/elm/Page/Profile.elm [hunk 2]
```diff
@@ -330,10 +333,19 @@ viewForm loggedIn isDisabled profile avatarS form =
                                 , value form.bio
                                 , disabled isDisabled
                                 , plcH "profile.edit.placeholders.bio"
+                                , maxlength 255
```
Comment (lucca65) [inline src/elm/Page/Profile.elm]: nice! +1

File: src/styles/main.css [hunk 3]
```diff
@@ -105,6 +105,10 @@
     @apply form-input h-12;
 }
 
+.input-counter {
+    @apply text-purple-100 uppercase font-bold float-right mt-1;
+}
+
```
Comment (altjohndev) [inline src/styles/main.css]: #156 #184

Is this the right way to use Tailwind?

Comment (lucca65) [reply src/styles/main.css]: it is! thanks @altjohndev

**Pull request merged**